app/gui/src/dashboard/appUtils.tsx

```diff
@@ -4,6 +4,8 @@ export const ORGANIZATION_NAME_MAX_LENGTH = 64
 
 export const OPEN_IDE_DEEPLINK = 'enso://'
 
+export const API_URL = $config.API_URL
```

Comment:
so is it `API_URL` or `AUTH_API_URL`? for `production` and `staging`  it is the same but e.g my personal envs are using different uris (staging cognito for auth and my api for other stuff)